### PR TITLE
list remote reference and get list of branches

### PR DESCRIPTION
**Pull request by anchal20**

### CODE REVIEW

File: tips.json [hunk 1]
```diff
@@ -52,6 +52,18 @@
       "tip": "git remote set-url origin <URL>"
   },
   {
+    "title": "Get list of all remote references",
+    "tip": "git remote || git remote show"
```
Comment (hemanth) [inline tips.json]: `||` ?

Comment (anchal20) [inline tips.json]: i wanted to show either of the two can be used, hence used this. Can you suggest something else?

Comment (hemanth) [inline tips.json]: Well, it could be `tips:[ ]` but then we need to modify the doxie config a bit.

Comment (stoeffel) [inline tips.json]: how about `tip` and `alternatives`?

Comment (hemanth) [inline tips.json]: ^ makes sense, BTW the pre-commit hook is not working for her, are we missing something ?

Comment (stoeffel) [inline tips.json]: Did you run `npm install`?

**Comment by hemanth** [comment]
:+1: Will merge this, as of now doxie is not configured to work with alternatives, will fix that soon.

**Comment by hemanth** [comment]
@anchal20 @stoeffel Added support for alternatives, makes sure that alternatives is an array. :+1:

**Pull request merged**